src/screens/CreatePost/index.js

```diff
@@ -882,35 +891,35 @@ export default CreatePost;
 const styles = StyleSheet.create({
   container: {
     flex: 1,
-    backgroundColor: '#fff',
+    backgroundColor: COLORS.white,
     position: 'relative'
   },
   input: {
-    backgroundColor: colors.lightgrey,
+    backgroundColor: COLORS.lightgrey,
     paddingVertical: 16,
     paddingHorizontal: 12,
     minHeight: 100,
     justifyContent: 'flex-start',
     overflow: 'scroll'
   },
   hastagText: {
-    color: colors.gray1,
+    color: COLORS.gray9,
     fontSize: 14,
     fontFamily: fonts.inter[400]
   },
   listText: {
-    color: colors.black,
+    color: COLORS.black,
     fontSize: 14,
     fontFamily: fonts.inter[400]
   },
   label: {
-    color: colors.black,
+    color: COLORS.black,
     fontFamily: fonts.inter[600],
     fontWeight: 'bold'
   },
   desc: {fontSize: 14, fontFamily: fonts.poppins[400]},
   labelButtonAddMedia: {
-    color: colors.black,
+    color: COLORS.black,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are good as they replace hardcoded color values with constants from a theme file which improves maintainability.
```diff
-    backgroundColor: '#fff',
+    backgroundColor: COLORS.white,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->